### PR TITLE
fix: 0 showing as collected fee amount

### PR DESCRIPTION
**Pull request by JackShort**
## Description
* fixes bug where 0s are showed for the amount when collecting LP position fee

_Linear ticket:_ https://linear.app/uniswap/issue/WEB-1906/fee-collection-from-pool-appearing-as-0-in-activity-feed

### Before
Screen Shot 2023-06-20 at 4 09 55 PM

### After
Screen Shot 2023-06-20 at 4 08 25 PM

### CODE REVIEW

**Review by zzmp — CHANGES_REQUESTED** [review]
Can you construct a unit test with mocked network responses (unit tests should not rely on network)?

**Comment by JackShort** [comment]
> Can you construct a unit test with mocked network responses (unit tests should not rely on network)?

we actually have this in `parseLocal` and tests and it tests all of the functions that

File: src/pages/Pool/PositionPage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        CurrencyAmount.fromRawAmount(USDC_MAINNET, '1224156977'),
+        CurrencyAmount.fromRawAmount(WETH9[1], '500807669662847869'),
+      ]
+    })
+
+    render(<PositionPage />)
+
+    const collectFeesButton = screen.queryByTestId('collect-fees-button') as HTMLButtonElement
+    expect(collectFeesButton).toBeInTheDocument()
+    expect(screen.getByText('Collect fees')).toBeInTheDocument()
```
Comment (zzmp) [inline src/pages/Pool/PositionPage.test.tsx:74]: You mock it so it renders actual numbers, but you should also assert on that: expect(...).toContain(actualNumbers)...

**Review by zzmp — CHANGES_REQUESTED** [review]
sorry last nit to get the tests up to prevent regressing

**Review by zzmp — APPROVED** [review]

**Review by vm — APPROVED** [review]

**Pull request merged**